File: app/src/main/java/com/example/marosu/secretchat/model/body/LoginBody.java
package com.example.marosu.secretchat.model.body;

import com.google.gson.annotations.SerializedName;

/**
 * Created by Marius-Andrei Rosu on 8/28/2017.
 */
public class LoginBody {
    @SerializedName("email")
    private String email;

    @SerializedName("password")
    private String password;

    public LoginBody(String email, String password) {
        this.email = email;
        this.password = password;
    }
}

File: app/src/main/java/com/example/marosu/secretchat/model/body/LoginBody.kt
package com.example.marosu.secretchat.model.body

import com.google.gson.annotations.SerializedName

/**
 * Created by Marius-Andrei Rosu on 8/29/2017.
 */
data class LoginBody(@SerializedName("username") val username: String,
                     @SerializedName("password") val password: String)